packages/rich-text/src/helpers/editor.ts

```diff
@@ -231,3 +233,17 @@ export const isInlineOrText = (node: Node) => {
   // either text or inline elements
   return Text.isText(node) || (Element.isElement(node) && INLINE_TYPES.includes(node.type));
 };
+
+export const focus = (editor: PlateEditor) => {
+  const x = window.scrollX;
+  const y = window.scrollY;
+
+  ReactEditor.focus(editor);
+
+  // Safari has issues with `editor.focus({ preventScroll: true })`, it ignores the option `preventScroll`
+  if (IS_SAFARI) {
+    setTimeout(function () {
+      window.scrollTo(x, y); // restore position
+    }, 0);
+  }
+};
```

Comment:
The first commit simply ignores Safari

 https://github.com/contentful/field-editors/pull/1006/commits/185b0da497195eae03206b95d107e1d0c0d9a5a6#diff-4b9fbdb58a22376a73c74bceb917c863f1c56d9e02c096332f77a63cbae48358R237-R242